### ISSUE TITLE
"x86_64-conda_cos6-linux-gnu/bin/ld: cannot find -lcrypto"

### ISSUE THREAD

**Issue by flopezo** [conda/conda#7578]
**I'm submitting a...**
  - [ ] bug report
  - [ ] feature request

## Current Behavior

I installed the [conda compiler tools](https://conda.io/docs/user-guide/tasks/build-packages/compiler-tools.html) and am trying to compile a third-party program:

```
conda install cmake curl libcurl gcc_linux-64 gxx_linux-64 gfortran_linux-64 libgcc-ng libgcc openmp openssl isl cloog mpc mpfr
source activate root
cmake -DINSTALL_PREFIX=/path/to/build ..
make -j MP
```
However, I'm getting the linkage errors below:

```
.../miniconda2/bin/../lib/gcc/x86_64-conda_cos6-linux-gnu/7.2.0/../../../../x86_64-conda_cos6-linux-gnu/bin/ld: cannot find -lcrypto
.../miniconda2/bin/../lib/gcc/x86_64-conda_cos6-linux-gnu/7.2.0/../../../../x86_64-conda_cos6-linux-gnu/bin/ld: cannot find -lcurl
.../miniconda2/bin/../lib/gcc/x86_64-conda_cos6-linux-gnu/7.2.0/../../../../x86_64-conda_cos6-linux-gnu/bin/ld: cannot find -lssl
```

### Steps to Reproduce

```

```

## Environment Information
<details open><summary><code>`conda info`</code></summary><p>

```

     active environment : None
       user config file : /home/username/.condarc
 populated config files : /home/username/.condarc
          conda version : 4.5.8
    conda-build version : not installed
         python version : 2.7.14.final.0
       base environment : /home/username/miniconda2  (writable)
           channel URLs : https://conda.anaconda.org/bioconda/linux-64
                          https://conda.anaconda.org/bioconda/noarch
                          https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/linux-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/linux-64
                          https://repo.anaconda.com/pkgs/pro/noarch
                          https://conda.anaconda.org/r/linux-64
                          https://conda.anaconda.org/r/noarch
          package cache : /home/username/miniconda2/pkgs
                          /home/username/.conda/pkgs
       envs directories : /home/username/miniconda2/envs
                          /home/username/.conda/envs
               platform : linux-64
             user-agent : conda/4.5.8 requests/2.18.4 CPython/2.7.14 Linux/3.10.0-862.3.3.el7.x86_64 centos/7 glibc/2.17
                UID:GID : 3269:50000
             netrc file : None
           offline mode : False
```
</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

```
==> /home/username/.condarc <==
channels:
  - bioconda
  - conda-forge
  - defaults
  - r
```
</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

```
# packages in environment at /home/username/miniconda2:
#
# Name                    Version                   Build  Channel
abyss                     2.1.0                h09b0a5c_0    bioconda
appdirs                   1.4.3                      py_1    conda-forge
argtable2                 2.13                          1    conda-forge
armadillo                 8.500.0         blas_openblash45cfd7e_202  [blas_openblas]  conda-forge
arpack                    3.5.0           blas_openblas_202  [blas_openblas]  conda-forge
asn1crypto                0.24.0                   py27_0    defaults
assembly-stats            1.0.1                         0    bioconda
backports                 1.0                      py27_1    conda-forge
backports.functools_lru_cache 1.5                      py27_0    conda-forge
backports_abc             0.5                      py27_0    conda-forge
bamtools                  2.4.1                         1    bioconda
bcftools                  1.8                           1    bioconda
beagle-lib                2.1.2                         7    bioconda
beast2                    2.4.5                         0    bioconda
bfc                       r181                          2    bioconda
binutils_impl_linux-64    2.28.1               had2808c_3    anaconda
binutils_linux-64         7.2.0               had2808c_27    anaconda
bioawk                    1.0                  ha92aebf_3    bioconda
bioconductor-biobase      2.38.0                 r3.4.1_0    bioconda
bioconductor-biocgenerics 0.24.0                 r3.4.1_0    bioconda
bioconductor-edger        3.20.7                 r3.4.1_0    bioconda
bioconductor-limma        3.34.9                 r3.4.1_0    bioconda
bioconductor-qvalue       2.10.0                 r3.4.1_0    bioconda
biopython                 1.70                np112py27_1    bioconda
blas                      1.0                         mkl    defaults
blast                     2.7.1               boost1.64_3    bioconda
blat                      36                            0    bioconda
boost                     1.64.0                   py27_4    conda-forge
boost-cpp                 1.64.0                        1    conda-forge
bowtie                    1.2.2            py27pl5.22.0_0    bioconda
bowtie2                   2.3.4.1          py27pl5.22.0_0    bioconda
bwa                       0.7.17               pl5.22.0_2    bioconda
bzip2                     1.0.6                         1    conda-forge
ca-certificates           2018.4.16                     0    conda-forge
cairo                     1.14.10                       0    conda-forge
capnproto                 0.6.1                         0    conda-forge
cd-hit                    4.6.8                         0    bioconda
certifi                   2018.4.16                py27_0    anaconda
cffi                      1.11.5           py27h9745a5d_0    defaults
chardet                   3.0.4            py27hfa10054_1    defaults
click                     6.7                        py_1    conda-forge
cloog                     0.18.1                        0    defaults
cmake                     3.12.0               h011004d_0    conda-forge
collectl                  4.0.4                pl5.22.0_3    bioconda
colorama                  0.3.9                     <pip>
colormath                 3.0.0                      py_2    conda-forge
conda                     4.5.8                    py27_1    conda-forge
conda-env                 2.6.0                h36134e3_1    defaults
crb-blast                 0.6.6                         0    bioconda
cryptography              2.2.2            py27h14c3975_0    defaults
curl                      7.60.0                        0    conda-forge
cycler                    0.10.0                   py27_0    conda-forge
cython                    0.28.2                   py27_0    conda-forge
dbus                      1.11.0                        0    conda-forge
decorator                 4.3.0                      py_0    conda-forge
diamond                   0.9.21                        1    bioconda
dlcpar                    1.0                      py27_0    bioconda
docopt                    0.6.2                    py27_0    conda-forge
emboss                    6.5.7                         4    bioconda
enum34                    1.1.6            py27h99a27e9_1    defaults
ete3                      3.1.1                     <pip>
exonerate                 2.4.0                         0    bioconda
expat                     2.2.5                         0    conda-forge
fasta3                    36.3.8                        2    bioconda
fastme                    2.1.5                         0    bioconda
fastool                   0.1.4                         2    bioconda
fastqc                    0.11.7               pl5.22.0_1    bioconda
fasttree                  2.1.10                        0    bioconda
font-ttf-ubuntu           0.83                 h8b1ccd4_0    defaults
fontconfig                2.12.6                        0    conda-forge
freetype                  2.8.1                         0    conda-forge
ftputil                   3.2                      py27_0    bioconda
functools32               3.2.3.2                  py27_2    conda-forge
future                    0.16.0                   py27_0    conda-forge
futures                   3.2.0            py27h7b459c0_0    defaults
gawk                      4.2.1                h14c3975_0    anaconda
gcc_impl_linux-64         7.2.0                habb00fd_3    anaconda
gcc_linux-64              7.2.0               h550dcbe_27    defaults
gclib                     0.0.1                         4    bioconda
genometools               1.2.1                    py27_0    bioconda
gettext                   0.19.8.1                      0    conda-forge
gffcompare                0.10.1                        0    bioconda
gffread                   0.9.9                         0    bioconda
gfortran_impl_linux-64    7.2.0                hdf63c60_3    defaults
gfortran_linux-64         7.2.0               h550dcbe_27    defaults
giflib                    5.1.4                         0    conda-forge
glib                      2.55.0                        0    conda-forge
gmap                      2018.03.25           pl5.22.0_0    bioconda
gmp                       6.1.2                         0    conda-forge
gnuplot                   5.0.3                         4    bioconda
gnutls                    3.5.17                        0    conda-forge
graphite2                 1.3.11                        0    conda-forge
graphviz                  2.38.0                        7    conda-forge
gsl                       2.1                           2    conda-forge
gst-plugins-base          1.8.0                         0    conda-forge
gstreamer                 1.8.0                         1    conda-forge
gxx_impl_linux-64         7.2.0                hdf63c60_3    anaconda
gxx_linux-64              7.2.0               h550dcbe_27    defaults
harfbuzz                  1.7.6                         0    conda-forge
hdf5                      1.8.17                       11    conda-forge
hisat2                    2.1.0            py27pl5.22.0_0    bioconda
hmmer                     3.1b2                         3    bioconda
hyphy                     2.3.12               h5466e78_1    bioconda
icu                       58.2                          0    conda-forge
idna                      2.6              py27h5722d68_1    defaults
infernal                  1.1.2                         0    bioconda
intel-openmp              2018.0.0                      8    defaults
ipaddress                 1.0.22                   py27_0    defaults
iqtree                    1.5.5                         1    bioconda
isl                       0.19                          0    conda-forge
java-jdk                  8.0.92                        1    bioconda
jellyfish                 2.2.6                         0    bioconda
jemalloc                  4.5.0                         0    bioconda
jinja2                    2.10                     py27_0    conda-forge
jpeg                      9c                   h470a237_0    conda-forge
kallisto                  0.44.0             hdf51.8.17_1    bioconda
kat                       2.3.1               boost1.64_2    bioconda
kmc                       3.0.1                noasmlib_2    bioconda
krb5                      1.14.6                        0    conda-forge
lapack                    3.6.0                         2    bioconda
lastz                     1.0.4                         1    bioconda
leveldb                   1.18                          0    conda-forge
libcurl                   7.60.0               h1ad7b7a_0    defaults
libdb                     6.1.26                        0    bioconda
libdeflate                0.8                           0    bioconda
libedit                   3.1.20170329         haf1bffa_0    conda-forge
libffi                    3.2.1                hd88cf55_4    defaults
libgcc                    7.2.0                h69d50b8_2    defaults
libgcc-ng                 7.2.0                hdf63c60_3    defaults
libgd                     2.2.5                         3    conda-forge
libgfortran               3.0.0                         1    defaults
libgfortran-ng            7.2.0                hdf63c60_3    defaults
libiconv                  1.15                          0    conda-forge
libidn11                  1.33                          0    conda-forge
libopenblas               0.2.20               h9ac9557_7    defaults
libpng                    1.6.34                        0    conda-forge
libssh2                   1.8.0                h5b517e9_2    conda-forge
libstdcxx-ng              7.2.0                hdf63c60_3    defaults
libtiff                   4.0.9                         0    conda-forge
libtool                   2.4.6                         0    conda-forge
libuuid                   1.0.3                         1    conda-forge
libuv                     1.20.3                        0    conda-forge
libwebp                   0.5.2                         7    conda-forge
libxcb                    1.13                          0    conda-forge
libxml2                   2.9.8                         0    conda-forge
libxslt                   1.1.29                        4    conda-forge
llvm-meta                 6.0.0                         0    conda-forge
lzstring                  1.0.3                    py27_0    conda-forge
m4                        1.4.18                        0    conda-forge
mafft                     7.313                         0    bioconda
make                      4.2.1                h470a237_0    conda-forge
maker                     2.31.9                        3    bioconda
mako                      1.0.7                      py_1    conda-forge
markdown                  2.6.11                     py_0    conda-forge
markupsafe                1.0                      py27_0    conda-forge
mash                      2.0                    gsl2.2_2    bioconda
mashmap                   2.0                    gsl2.2_1    bioconda
matplotlib                2.1.0            py27h09aba24_0    defaults
mcl                       14.137                        2    bioconda
miniasm                   0.2                           0    bioconda
minimap2                  2.11                 ha92aebf_0    bioconda
mkl                       2018.0.2                      1    defaults
mkl_fft                   1.0.2                    py27_0    conda-forge
mkl_random                1.0.1                    py27_0    conda-forge
mmtf-python               1.0.2                    py27_0    bioconda
mpc                       1.1.0                hb705a9b_6    conda-forge
mpfr                      4.0.1                h16a7912_0    conda-forge
mpi                       1.0                     openmpi    conda-forge
mpi4py                    2.0.0                    py27_2    defaults
mpich2                    1.4.1p1                       0    defaults
msgpack-python            0.5.6            py27h2d50403_2    conda-forge
multiqc                   1.5                      py27_0    bioconda
mysql                     5.7.20                        0    conda-forge
mysql-connector-c         6.1.6                         2    bioconda
mysql-python              1.2.5                    py27_0    defaults
mysqlclient               1.3.10                   py27_0    bioconda
ncurses                   6.1                           0    conda-forge
nettle                    3.3                           0    conda-forge
networkx                  2.0                      py27_1    conda-forge
nlopt                     2.4.2                    py27_2    conda-forge
numpy                     1.12.1           py27h9378851_1    defaults
numpy-base                1.14.3           py27h0ea5e3f_1    defaults
ocl-icd                   2.2.9                h470a237_4    conda-forge
olefile                   0.45.1                   py27_0    conda-forge
openblas                  0.2.20                        7    conda-forge
openjdk                   8.0.121                       1    defaults
openmp                    6.0.0                         1    conda-forge
openmpi                   3.1.0                h26a2512_3    conda-forge
openssl                   1.0.2o                        0    conda-forge
paml                      4.9                           1    bioconda
pandoc                    2.2.1                         0    conda-forge
pango                     1.40.14                       0    conda-forge
parafly                   r2013_01_21                   1    bioconda
parallel                  20170422             pl5.22.0_0    bioconda
pcre                      8.39                          0    conda-forge
perl                      5.22.0.1                      0    conda-forge
perl-aceperl              1.92                          0    bioconda
perl-acme-damn            0.08                 pl5.22.0_0    bioconda
perl-algorithm-munkres    0.08                          0    bioconda
perl-app-cpanminus        1.7043               pl5.22.0_0    bioconda
perl-appconfig            1.71                          0    bioconda
perl-archive-tar          2.18                 pl5.22.0_2    bioconda
perl-array-compare        2.11                          0    bioconda
perl-bio-asn1-entrezgene  1.72                          1    bioconda
perl-bio-featureio        1.6.905                       0    bioconda
perl-bio-phylo            0.58                          0    bioconda
perl-bio-samtools         1.43                          0    bioconda
perl-bioperl              1.6.924              pl5.22.0_7    bioconda
perl-bioperl-core         1.6.924              pl5.22.0_2    bioconda
perl-bioperl-run          1.006900                      2    bioconda
perl-bit-vector           7.4                           0    bioconda
perl-carp                 1.38                 pl5.22.0_0    bioconda
perl-cgi                  4.22                          3    bioconda
perl-class-inspector      1.28                          0    bioconda
perl-class-method-modifiers 2.11                          1    bioconda
perl-clone                0.38                          0    bioconda
perl-common-sense         3.74                          0    bioconda
perl-compress-raw-bzip2   2.069                         1    bioconda
perl-compress-raw-zlib    2.069                         3    bioconda
perl-convert-binary-c     0.78                          0    bioconda
perl-convert-binhex       1.125                         0    bioconda
perl-crypt-rc4            2.02                          0    bioconda
perl-data-dumper          2.161                pl5.22.0_0    bioconda
perl-data-stag            0.14                          0    bioconda
perl-db-file              1.835                         5    bioconda
perl-dbd-mysql            4.033                         2    bioconda
perl-dbd-pg               3.5.3                         0    bioconda
perl-dbd-sqlite           1.50                          1    bioconda
perl-dbi                  1.641                pl5.22.0_0    bioconda
perl-devel-globaldestruction 0.13                          1    bioconda
perl-digest-hmac          1.03                 pl5.22.0_1    bioconda
perl-digest-perl-md5      1.9                           0    bioconda
perl-email-date-format    1.005                         0    bioconda
perl-encode-locale        1.05                 pl5.22.0_4    bioconda
perl-error                0.17024                       0    bioconda
perl-exporter             5.72                 pl5.22.0_0    bioconda
perl-exporter-tiny        0.042                         1    bioconda
perl-extutils-makemaker   7.24                 pl5.22.0_1    bioconda
perl-file-copy-recursive  0.38                          1    bioconda
perl-file-find-rule       0.34                          1    bioconda
perl-file-listing         6.04                          0    bioconda
perl-file-sharedir        1.102                         3    bioconda
perl-file-sharedir-install 0.10                          3    bioconda
perl-file-slurp-tiny      0.004                         0    bioconda
perl-file-sort            1.01                 pl5.22.0_1    bioconda
perl-file-which           1.20                          0    bioconda
perl-font-afm             1.20                          0    bioconda
perl-forks                0.36                 pl5.22.0_2    bioconda
perl-gd                   2.56                          5    bioconda
perl-graph                0.9704                        0    bioconda
perl-graphviz             2.20                          1    bioconda
perl-html-element-extended 1.18                          0    bioconda
perl-html-entities-numbered 0.04                          0    bioconda
perl-html-formatter       2.14                          0    bioconda
perl-html-parser          3.72                 pl5.22.0_1    bioconda
perl-html-tableextract    2.13                          0    bioconda
perl-html-tagset          3.20                 pl5.22.0_1    bioconda
perl-html-tidy            1.56                          1    bioconda
perl-html-tree            5.03                          0    bioconda
perl-html-treebuilder-xpath 0.14                          0    bioconda
perl-http-cookies         6.01                          0    bioconda
perl-http-daemon          6.01                          0    bioconda
perl-http-date            6.02                 pl5.22.0_1    bioconda
perl-http-message         6.11                          0    bioconda
perl-http-negotiate       6.01                          0    bioconda
perl-image-info           1.38                          0    bioconda
perl-image-size           3.300                         0    bioconda
perl-inline               0.80                          0    bioconda
perl-inline-c             0.76                          0    bioconda
perl-io-all               0.86                          0    bioconda
perl-io-compress          2.069                pl5.22.0_2    bioconda
perl-io-html              1.001                pl5.22.0_1    bioconda
perl-io-prompt            0.997004             pl5.22.0_0    bioconda
perl-io-sessiondata       1.03                          0    bioconda
perl-io-socket-ssl        2.024                         0    bioconda
perl-io-string            1.08                 pl5.22.0_1    bioconda
perl-io-stringy           2.111                         0    bioconda
perl-io-tty               1.12                          0    bioconda
perl-io-zlib              1.10                          1    bioconda
perl-ipc-run              0.94                          0    bioconda
perl-jcode                2.07                          0    bioconda
perl-json                 2.90                          1    bioconda
perl-json-xs              2.34                          0    bioconda
perl-libwww-perl          6.15                          0    bioconda
perl-libxml-perl          0.08                          0    bioconda
perl-list-moreutils       0.15                 pl5.22.0_0    bioconda
perl-lwp-mediatypes       6.02                 pl5.22.0_1    bioconda
perl-lwp-protocol-https   6.06                          2    bioconda
perl-lwp-simple           6.15                          3    bioconda
perl-mailtools            2.14                          0    bioconda
perl-math-cdf             0.1                           3    bioconda
perl-math-derivative      0.04                          0    bioconda
perl-math-random          0.72                          0    bioconda
perl-math-spline          0.02                          0    bioconda
perl-mime-base64          3.15                 pl5.22.0_0    bioconda
perl-mime-lite            3.030                         0    bioconda
perl-mime-tools           5.507                         0    bioconda
perl-mime-types           2.12                          0    bioconda
perl-mldbm                2.05                          0    bioconda
perl-module-build         0.4224               pl5.22.0_0    bioconda
perl-module-runtime       0.014                         1    bioconda
perl-moo                  2.001000                      1    bioconda
perl-net-http             6.09                          0    bioconda
perl-net-ssleay           1.84                 pl5.22.0_0    bioconda
perl-ntlm                 1.09                          1    bioconda
perl-number-compare       0.03                 pl5.22.0_1    bioconda
perl-ole-storage_lite     0.19                          0    bioconda
perl-parse-recdescent     1.967013                      0    bioconda
perl-pathtools            3.73                          0    bioconda
perl-pdf-api2             2.025                         2    bioconda
perl-pegex                0.61                          0    bioconda
perl-perl-unsafe-signals  0.03                          0    bioconda
perl-perlio-gzip          0.19                 pl5.22.0_2    bioconda
perl-postscript           0.06                          0    bioconda
perl-role-tiny            2.000001                      1    bioconda
perl-scalar-list-utils    1.45                          2    bioconda
perl-set-scalar           1.29                          0    bioconda
perl-soap-lite            1.19                          0    bioconda
perl-sort-naturally       1.03                          0    bioconda
perl-spreadsheet-parseexcel 0.65                          0    bioconda
perl-spreadsheet-writeexcel 2.40                          0    bioconda
perl-statistics-descriptive 3.0612                        0    bioconda
perl-sub-exporter-progressive 0.001011                      1    bioconda
perl-svg                  2.64                          0    bioconda
perl-svg-graph            0.02                          0    bioconda
perl-sys-sigaction        0.23                 pl5.22.0_0    bioconda
perl-task-weaken          1.04                          0    bioconda
perl-template-toolkit     2.26                          0    bioconda
perl-termreadkey          2.32                          0    bioconda
perl-test-leaktrace       0.16                 pl5.22.0_0    bioconda
perl-test-more            1.001002             pl5.22.0_0    bioconda
perl-test-pod             1.51                          0    bioconda
perl-text-glob            0.11                 pl5.22.0_0    bioconda
perl-text-soundex         3.05                          0    bioconda
perl-threaded             5.22.0                       10    bioconda
perl-tie-ixhash           1.23                 pl5.22.0_1    bioconda
perl-timedate             2.30                          0    bioconda
perl-tree-dag_node        1.29                          0    bioconda
perl-type-tiny            1.000005                      0    bioconda
perl-unicode-map          0.112                         0    bioconda
perl-uri                  1.71                 pl5.22.0_1    bioconda
perl-want                 0.29                 pl5.22.0_0    bioconda
perl-www-robotrules       6.02                          0    bioconda
perl-xml-dom              1.45                          0    bioconda
perl-xml-dom-xpath        0.14                          0    bioconda
perl-xml-filter-buffertext 1.01                          0    bioconda
perl-xml-libxml           2.0124                        0    bioconda
perl-xml-libxslt          1.94                          0    bioconda
perl-xml-namespacesupport 1.11                          0    bioconda
perl-xml-parser           2.44                          4    bioconda
perl-xml-regexp           0.04                          0    bioconda
perl-xml-sax              0.99                          0    bioconda
perl-xml-sax-base         1.08                          0    bioconda
perl-xml-sax-expat        0.51                          0    bioconda
perl-xml-sax-writer       0.56                          0    bioconda
perl-xml-simple           2.22                          0    bioconda
perl-xml-twig             3.49                          0    bioconda
perl-xml-writer           0.625                         0    bioconda
perl-xml-xpath            1.33                          0    bioconda
perl-xml-xpathengine      0.14                          0    bioconda
perl-xsloader             0.22                 pl5.22.0_0    bioconda
perl-yaml                 1.24                          0    bioconda
perl-yaml-libyaml         0.66                 pl5.22.0_0    bioconda
phyphy                    0.4.2                     <pip>
pigz                      2.3.4                         0    conda-forge
pillow                    5.1.0                    py27_0    conda-forge
pilon                     1.22                     py27_0    bioconda
pip                       9.0.3                    py27_0    conda-forge
pixman                    0.34.0                        2    conda-forge
plyvel                    1.0.4                     <pip>
postgresql                10.3                     py27_0    conda-forge
prank                     v.170427                      0    bioconda
pycosat                   0.6.3            py27ha4109ae_0    defaults
pycparser                 2.18             py27hefa08c5_1    defaults
pyopenssl                 17.5.0           py27hcee3be0_0    defaults
pyparsing                 2.2.0                    py27_0    conda-forge
pyqt                      5.6.0                    py27_5    conda-forge
pysocks                   1.6.8                    py27_0    defaults
python                    2.7.14              ha6fc286_23    defaults
python-dateutil           2.3                      py27_0    bioconda
pytools                   2018.5.2                   py_0    conda-forge
pytz                      2018.4                     py_0    conda-forge
pyyaml                    3.12                     py27_1    conda-forge
qt                        5.6.2                         7    conda-forge
r-assertthat              0.2.0                  r3.4.1_0    conda-forge
r-base                    3.4.1                         4    conda-forge
r-cli                     1.0.0                  r3.4.1_0    conda-forge
r-cluster                 2.0.6                  r3.4.1_0    r
r-colorspace              1.3_2                  r3.4.1_0    conda-forge
r-crayon                  1.3.4                  r3.4.1_0    conda-forge
r-dichromat               2.0_0                  r3.4.1_0    conda-forge
r-digest                  0.6.15                 r3.4.1_0    conda-forge
r-ggplot2                 2.2.1                  r3.4.1_0    conda-forge
r-glue                    1.2.0                  r3.4.1_0    conda-forge
r-gtable                  0.2.0                  r3.4.1_0    conda-forge
r-labeling                0.3                    r3.4.1_0    conda-forge
r-lattice                 0.20_34                r3.4.1_0    conda-forge
r-lazyeval                0.2.1                  r3.4.1_0    conda-forge
r-locfit                  1.5_9.1                r3.4.1_0    conda-forge
r-magrittr                1.5                    r3.4.1_0    conda-forge
r-mass                    7.3_48                 r3.4.1_0    conda-forge
r-munsell                 0.5.0                    r341_0    conda-forge
r-pillar                  1.2.2                    r341_0    conda-forge
r-plyr                    1.8.4                  r3.4.1_0    conda-forge
r-r6                      2.2.2                  r3.4.1_0    conda-forge
r-rcolorbrewer            1.1_2                  r3.4.1_0    conda-forge
r-rcpp                    0.12.15                r3.4.1_0    conda-forge
r-reshape2                1.4.3                  r3.4.1_0    conda-forge
r-rlang                   0.2.1                    r341_0    conda-forge
r-scales                  0.5.0                  r3.4.1_0    conda-forge
r-stringi                 1.1.7                  r3.4.1_1    conda-forge
r-stringr                 1.3.0                  r3.4.1_0    conda-forge
r-tibble                  1.4.2                  r3.4.1_0    conda-forge
r-utf8                    1.1.3                  r3.4.1_0    conda-forge
r-viridislite             0.3.0                  r3.4.1_0    conda-forge
raxml                     8.2.10                        0    bioconda
readline                  7.0                  haf1bffa_1    conda-forge
recon                     1.08                          0    bioconda
rename                    1.600                pl5.22.0_0    bioconda
repeatscout               1.0.5                         0    bioconda
reportlab                 3.4.0                    py27_0    defaults
requests                  2.18.4           py27hc5b0589_1    defaults
rhash                     1.3.4                         0    conda-forge
ruamel_yaml               0.15.35          py27h14c3975_1    defaults
ruby                      2.2.3                         0    bioconda
samtools                  1.9                  h46bd0b3_0    bioconda
scipy                     1.1.0            py27hfc37229_0    defaults
seqkit                    0.8.0                         1    bioconda
seqmagick                 0.6.1                    py27_0    bioconda
seqtk                     1.2                           1    bioconda
setuptools                39.0.1                   py27_0    defaults
sga                       0.10.15         bamtools2.4.1_2    bioconda
simplejson                3.8.1                    py27_0    bioconda
singledispatch            3.4.0.3                  py27_0    conda-forge
sip                       4.18                     py27_1    conda-forge
six                       1.11.0           py27h5f960f1_1    defaults
slclust                   02022010                      2    bioconda
snap                      2013_11_29                    0    bioconda
snappy                    1.1.7                         1    conda-forge
spades                    3.11.1          py27_zlib1.2.11_1    bioconda
sparsehash                2.0.2                         0    bioconda
spectra                   0.0.11                     py_0    conda-forge
sqlite                    3.20.1               hb898158_2    anaconda
sra-tools                 2.8.2                         0    bioconda
stringtie                 1.3.3                    py27_3    bioconda
subprocess32              3.5.1                    py27_0    conda-forge
superlu                   5.2.1           blas_openblas_202  [blas_openblas]  conda-forge
tbb                       2018_20171205                 0    conda-forge
termcolor                 1.1.0                     <pip>
tidyp                     1.04                          0    bioconda
tk                        8.6.7                hc745277_3    defaults
tldr                      0.4.4                     <pip>
tornado                   5.0.2                    py27_0    conda-forge
transdecoder              5.2.0                pl5.22.0_0    bioconda
trf                       4.09                          0    bioconda
trimmomatic               0.36                          5    bioconda
trinity                   2.5.1                         1    bioconda
trinotate                 3.1.1                pl5.22.0_0    bioconda
trnascan-se               2.0                  pl5.22.0_0    bioconda
ucsc-pslcdnafilter        357                           1    bioconda
unicodecsv                0.14.1                   py27_0    conda-forge
urllib3                   1.22             py27ha55213b_0    defaults
wheel                     0.31.0                   py27_0    defaults
xmltodict                 0.9.2                    py27_0    bioconda
xorg-fixesproto           5.0                           1    conda-forge
xorg-inputproto           2.3.2                         1    conda-forge
xorg-kbproto              1.0.7                         1    conda-forge
xorg-libx11               1.6.5                         0    conda-forge
xorg-libxau               1.0.8                         3    conda-forge
xorg-libxdmcp             1.1.2                         3    conda-forge
xorg-libxext              1.3.3                         2    conda-forge
xorg-libxfixes            5.0.3                         2    conda-forge
xorg-libxi                1.7.8                         2    conda-forge
xorg-libxtst              1.2.3                         0    conda-forge
xorg-recordproto          1.14.2                        1    conda-forge
xorg-xextproto            7.3.0                         1    conda-forge
xorg-xproto               7.0.31                        6    conda-forge
xz                        5.2.3                         0    conda-forge
yaml                      0.1.7                had09818_2    defaults
zlib                      1.2.11               h470a237_3    conda-forge
```
</p></details>

**Comment by nehaljwani** [comment]
@flopezo Try passing `-Wl,--sysroot=/` to LDFLAGS ?

**Comment by mingwandroid** [comment]
Why would that help? Our sysroot is not at /

Try -L$CONDA_PREFIX/lib -Wl,-rpath,$CONDA_PREFIX/lib -Wl,rpath-link,$CONDA-PREFIX/lib

You should check some some of our cmake using recipes though. You haven't passed nearly enough options.

Also you haven't given is anything we can work with. What third party program? How are we meant to reproduce this?

**Comment by nehaljwani** [comment]
From the output, it seemed that OP is trying to link against openssl and curl libraries and has probably been installed via the system package manager. (Assuming OP is just trying to use the Anaconda Compilers for normal compilation and not a conda package)

**Comment by mingwandroid** [comment]
Given the conda install command line I disagree.

**Comment by nehaljwani** [comment]
Oh right. Sorry, didn't notice that. @flopezo please ignore my comment.

**Comment by meishiue** [comment]
Hello,
I have the same problem here with an other flag : -lz 
What I have in my terminal is "/home/msk/anaconda3/bin/x86_64-conda_cos6-linux-gnu-ld: cannot find -lz"
I have this problem with a program named meme-suite (http://meme-suite.org). When i try compile files with Makefile it can't finish because of this error.

I installed ld with "conda install -c conda-forge binutils"
Do you have any idea ?
